server/ast/func_expr.go

```diff
@@ -54,12 +54,12 @@ func nodeFuncExpr(node *tree.FuncExpr) (*vitess.FuncExpr, error) {
 	}
 	var distinct bool
 	switch node.Type {
-	case 0:
+	case 0, tree.AllFuncType:
 		distinct = false
 	case tree.DistinctFuncType:
 		distinct = true
-	case tree.AllFuncType:
-		return nil, fmt.Errorf("function spec is not yet supported")
+	default:
+		return nil, fmt.Errorf("unknown function spec type %d", node.Type)
```

Comment:
Wasn't aware that `ALL` is the opposite of `DISTINCT`. When I originally tried looking for the `ALL` keyword, all I found was [this page](https://www.postgresql.org/docs/15/functions-comparisons.html#id-1.5.8.30.17), which didn't seem like something we'd yet support.

Reply:
Yup, ALL is implicit but you can also specify it manually.